### DISCUSSION TITLE
Planned syntax changes?

### DISCUSSION THREAD

**Discussion by kee-oth** [joe-bell/cva#89] [Q&A] [answered]
Hi!

Considering cva hasn't hit v1.0.0 yet, I'm just curious if there are any major (or minor) syntax changes planned. I'm just interested in using it for a project but I'm concerned about breaking changes as cva marches towards v1.0.0

Thanks!

**Comment by joe-bell** [comment, marked as answer]
None planned just yet!

**Reply by kee-oth** [reply]
Thanks for the quick response!
